src/compiler/diagnosticMessages.json

```diff
@@ -7548,5 +7548,13 @@
     "The value '{0}' cannot be used here.": {
         "category": "Error",
         "code": 18050
+    },
+    "Convert typedef to type": {
+        "category": "Message",
+        "code": 18051
+    },
+    "Convert all typedefs to types": {
+        "category": "Message",
+        "code": 18052

```

Comment:
I think these should be in the 8000 block (see the codes used for the other diagnostics added in the part of the suggestionDiagnostics file you're modifying.

Reply:
fixed